### ISSUE TITLE
Wont accept large matrix sizes

### ISSUE THREAD

**Issue by SpudGB** [marcmerlin/FastLED_NeoMatrix#5]
Not sure if this is the right place so...
Using Marc's library I'm able to get scrolling text to work across my (self constructed) matrix but only to a point...

Once I enter a width of more than 34 pixels one (or more) library's fail.

My matrix is 37*8 (basic text scroller)  and I get back (serial output) either Framebuffer_GFX:begin Width 37 Height 7688 Num Pixels 1094851080

or 
Framebuffer_Framebuffer_ 

then hangs...
I've tried to identify the issue within Adafruit_GFX.h and Fastled_Neomatrix.h but cannot find why this error occurs.

**Comment by marcmerlin** [comment]
Ok, your question is a big vague in details.
The biggest Matrix I've tried was 240x320, which was a TFT screen.
I believe the code does handle both sides being bigger than 256 (up to 65536), and the total number of pixels being up to 2^32, but honestly testing has been super light because even 320x240 does not fit on any of my microcontrollers but one (and there there is not enough memory left for the generation code).
Note that I've only tested Framebuffer::GFX (and the underlying Adafruit::GFX) to those sizes. For FastLED::NeoMatrix, my largest array was 64x64.
I have to ask, how are you having a height of 7688 with neopixels? My guess is that you don't, because that many neopixels would be literally 4 million amps if you power them at 5V.
